### PR TITLE
Add 5 new units and one new qk

### PR DESCRIPTION
**Pull request by mgberg**
This includes:
- 1 new quantity kind (`quantitykind:SpecificModulus`)
- 5 new units (`unit:GigaPA-CentiM3-PER-GM`, `unit:KiloM2-PER-SEC2`, `unit:MicroM-PER-SEC`, `unit:MilliJ-PER-M2`, `unit:OHM-CentiM`)
- Modifying the quantity kind of 1 unit (`unit:M2-PER-SEC2`)

### CODE REVIEW

**Review by steveraysteveray — CHANGES_REQUESTED** [review]
We only allow 1 rdfs:label per language tag. Could you change the "meter" variants to #en-us? Thanks.

**Comment by steveraysteveray** [comment]
Sorry, meant \@en-us

**Comment by mgberg** [comment]
> Sorry, meant @en-us

Whoops, I created @en and @en-us labels but forgot to change the language tag of the en-us ones.

File: vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl [hunk 1]
```diff
@@ -20080,6 +20080,19 @@ quantitykind:SpecificInternalEnergy
   rdfs:seeAlso quantitykind:SpecificGibbsEnergy ;
   rdfs:seeAlso quantitykind:SpecificHelmholtzEnergy ;
 .
+quantitykind:SpecificModulus
+  a qudt:QuantityKind ;
+  dcterms:description """<p><b>Specific modulus</b> is a <a href=\"/wiki/Materials_property\" class=\"mw-redirect\" title=\"Materials property\">materials property</a> consisting of the <a href=\"/wiki/Elastic_modulus\" title=\"Elastic modulus\">elastic modulus</a> per mass <a href=\"/wiki/Density\" title=\"Density\">density</a> of a material. It is also known as the <b>stiffness to weight ratio</b> or <b>specific stiffness</b>. High specific modulus materials find wide application in <a href=\"/wiki/Aerospace\" title=\"Aerospace\">aerospace</a> applications where minimum structural <a href=\"/wiki/Weight\" title=\"Weight\">weight</a> is required. The <a href=\"/wiki/Dimensional_analysis\" title=\"Dimensional analysis\">dimensional analysis</a> yields units of distance squared per time squared. The equation can be written as:
```
Comment (steveraysteveray) [inline vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl:20085]: Actually, the dcterms:description kind of leaves me hanging at the very end. Perhaps delete the last sentence? The reader can still consult the latexDefinition...

**Review by steveraysteveray — CHANGES_REQUESTED** [review]

File: vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl (hunk 1 above)
Comment (mgberg) [reply vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl:20085]: Apparently the equation didn't copy correctly. But you're right, the latex definition has it. Fixed.

**Review by steveraysteveray — APPROVED** [review]

**Pull request merged**